index.d.ts

```diff
@@ -457,4 +447,23 @@ declare module 'gqutils' {
 		arg: (arg: any, opts?: string[] | {pick?: string[]}) => string;
 		toGqlArg: typeof toGqlArg;
 	}
+
+	class GqlApi<FragmentsMap = gqlFragmentMap, EnumsMap = gqlEnumMap> extends Gql<gqlFragmentMap, gqlEnumMap> {
+		constructor(api: _cacheOpts & ApiInput);
+
+		/** **NOTE**: Override this method to use your own http client */
+		static postRequest(url: string, opts: {body: any} & Omit<RequestOptions, 'endpoint'>): Promise<response>;
+	}
+
+	class GqlSchema<FragmentsMap = gqlFragmentMap, EnumsMap = gqlEnumMap> extends Gql<gqlFragmentMap, gqlEnumMap> {
```

Comment:
Types passed should not be default but the input
```suggestion
	class GqlSchema<FragmentsMap = gqlFragmentMap, EnumsMap = gqlEnumMap> extends Gql<FragmentsMap, EnumsMap> {
```